docs/operators_and_staking/staking.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Staking guide
+sidebar_position: 2
+description: Staking guide
+keywords:
+- Staking guide
+- Staking
+---
+
+### Select an operator to nominate 
+
+:::note
+Three important factors to pay attention to are `minimumNominatorStake`, `nominationTax`, and `status`.
+:::
+
+1. Visit PolkadotJS **[Network Explorer](https://polkadot.js.org/apps/#/explorer)**.
+2. Go to `Developer` -> `Chain state`.
+
+ ![Staking-16](/img/doc-imgs/operators-staking/Staking-16.png)
+3. Select `domains` under `selected state query` and choose `operators`, exclude `option` and click on `+` to run the query.
+4. Browse the list of available operators, make sure the status is `Registered` and that `minimumNominatorStake` is lower than your staking amount. 
+
+ ![Staking-17](/img/doc-imgs/operators-staking/Staking-17.png)
+
+:::note
+Number 17 on the screenshot above corresponds to `operatorID`.
+:::
+
+### Operator Nomination using [Polkadot.js](https://polkadot.js.org/)
+
+Any account can nominate any existing **operator** with at least a minimum nominator stake set by the **operator**. 
+
+:::info
+The only staking mechanism available to non-operators is nominating an operator.
+:::
+
+1. Make sure to select the correct network at the top-left corner. 
+2. Select the account you want to use in `using the selected account`.
+3. Select `domains` under `submit the following extrinsic` and choose `nominateOperator(operatorID, amount)` in the dropdown.
+4. Set an `operatorID` - in the example, it's set to **1**, 18 zeros are added since the field is of `u128` type.
```

Comment:
The 18 zeroes bit worked well as a note on the operators page would be nice to make it consistent here. Same comment about providing a 1 with 18 zeroes for convenience.

Reply:
Good idea 👍